conjure-codegen/src/context.rs

```diff
@@ -886,6 +886,8 @@ impl Context {
             | "typeof" | "unsized" | "virtual" | "yield" => true,
             // weak keywords
             "union" | "dyn" => true,
+            // builder pattern methods
+            "build" => true,
```

Comment:
We'll also want to cover `new` and `builder` I think to cover those static methods: https://docs.rs/conjure-codegen/latest/conjure_codegen/example_types/product/struct.SetExample.html